public/css/footer.css

```diff
@@ -0,0 +1,105 @@
+/***************** Footer Style ****************/
```

Comment:
for this file wait until this [pull request](https://github.com/FACG5/chat-it-out/pull/39) get merged and then get the latest update from the master then push it again so the footer.css will be sync with the changes that requested in the pull request mentioned above